backend-opal/src/server/watchLists.js

```diff
@@ -1,23 +1,16 @@
 const router = require('express').Router();
 const authService = require('./common/authorizationService');
-
-const data = [
-  {
-    id: 1,
-    userId: 1,
-    name: 'My Watchlist 1',
-    symbols: ['MSFT', 'AMZN', 'AAPL', 'FB']
-  }
-];
+const WatchListsData = require('./watchListService');
 
 function checkRequestBody(body) {
   const bodyKey = ['userId', 'name'];
   const result = bodyKey.filter(key => !(key in body));
   return result;
 }
 
-router.get('/', function(req, res) {
+router.get('/:UserId', async (req, res) => {
```

Comment:
@martinupset don't need the userId in the route

Reply:
ok